### DISCUSSION TITLE
Alert groups page jumps unpleasantly due to LoadingPlaceholder

### DISCUSSION THREAD

**Discussion by josephharding** [grafana/grafana#53792] [Alerting]
![ApplicationFrameHost_N1jSVoDUWs](https://user-images.githubusercontent.com/816137/184926830-e2d2d878-8632-45ea-b57d-bb9382e36d56.gif)
(above is a gif, annoying behavior is shown every 6 seconds)

I use the Alert groups page as my primary dashboard for viewing the state of my systems because of the powerful "Custom group by" functionality, however I find the constant page jump being caused by the LoadingPlaceholder spinner to be really irritating. It would be nice if the element's parent was fixed height so the contents of the entire page wasn't constantly shifting around.

Currently using Grafana 9.0.7

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
